pkg/cri/resource-manager/policy/flags.go

```diff
@@ -163,11 +165,28 @@ func (cs *ConstraintSet) parseCPUQuantity(value string) error {
 }
 
 func (cs *ConstraintSet) parseCPUFromCgroup(dir string) error {
-	cpusetDir := cgroups.Cpuset.Path()
-	if !strings.HasPrefix(dir, cpusetDir+"/") {
-		dir = filepath.Join(cpusetDir, dir)
+	ifExists := func(path string) string {
+		if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
+			return ""
+		}
+		return path
+	}
+	path := ""
+	if dir[0] == '/' {
```

Comment:
BTW, I think crashes if called with `dir := ""`...

Reply:
That crash happens already on `parseCPU()`. I think it makes to fix that in this same commit. Working on it.

Reply:
Fixed in both places. (Here it is dead code, but good to have if later on called with an empty string.)